hivemind/utils/grpc.py

```diff
@@ -46,14 +46,16 @@ def serialize_torch_tensor(tensor: torch.Tensor, compression_type=CompressionTyp
             size=tensor.shape,
             dtype='clamped_float32',
             requires_grad=tensor.requires_grad)
-    else:
+    elif compression_type == CompressionType.NONE:
         array = tensor.numpy()
         proto = runtime_pb2.Tensor(
             compression=compression_type,
             buffer=array.tobytes(),
             size=array.shape,
             dtype=array.dtype.name,
             requires_grad=tensor.requires_grad)
+    else:
+        raise ValueError(f"Unknown compression type: {compression_type}")
```

Comment:
rationale: if i specify an invalid compression scheme, i don't want the code to *silently* disable compression.